test/specs/lit.ts

```diff
@@ -79,6 +88,17 @@ test('lit-css', (t) => {
           color yellow
           background url("data:image/gif;base64,R0lGO")
         }
+        .cls {
+          &:before {
+            content: '';
+            position: absolute;
+            border-width: 0;
+          }
+          &:after {
```

Comment:
Let here be just `:after`, 'cause it's not necessary to write `&` in selectors in fela in most cases.

Reply:
better uncomment your part?

Reply:
yeah, the next, concatenated `.class:pseudo`  just how nasty minifiers breaked the code.